src/selectors/metadata.js

```diff
@@ -24,6 +23,18 @@ const runCommandTemplates = {
   task: (name) => `kedro run --to-nodes ${name}`,
 };
 
+/**
+ * Unwrap parameters by extracting the first object
+ *  @param {?object} parameters The parameters
+ *  @returns {object} The unwrapped parameters
+ */
+const unwrapParameters = (parameters) => {
```

Comment:
As per above, I would rename this function as `destructuredParameters`, or anything more clear other than unwrap